app/assets/scripts/config/production.js

```diff
@@ -9,5 +9,6 @@ module.exports = {
   api: 'http://openroads-vn-api.herokuapp.com',
   mbToken: 'pk.eyJ1Ijoib3BlbnJvYWRzIiwiYSI6InJ0aUQ2N3MifQ.R3hdFqriZr6kEUr-j_FYpg',
   editorUrl: '//orma.github.io/openroads-vn-iD/',
-  roadNetTileLayerUrl: 'http://50.16.162.86/dashboard/{z}/{x}/{y}.png'
+  roadNetTileLayerUrl: 'http://50.16.162.86/dashboard/{z}/{x}/{y}.png',
+  s3Bucket: 'https://s3.amazonaws.com/openroads-vn-dumps/by-province-name/'
```

Comment:
Last comment on this config: `s3Bucket` is very vague, and a project can reasonably have multiple ones, and technically this is just a base URL rather than a bucket name (which may confuse future devs, including future-us). Maybe make it `provinceDumpBaseURL` or something like that?